cli/jobs/pipelines/automl/cli-automl-classification-task-bankmarketing-pipeline/components/component_register_model.yaml

```diff
@@ -0,0 +1,24 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: register_model
+display_name: Register Model
+version: 1
+type: command
+inputs:
+  model_input_path:
+    type: uri_folder
+  model_base_name:
+    type: string
+  model_name_suffix: # Set negative to use epoch_secs
```

Comment:
What is this note for?

Reply:
If it is set -ve we use epochs to add suffix to model name 
else we use whatever is provided by the user.